app/javascript/hooks/mutations/shared_accesses/useDeleteSharedAccess.jsx

```diff
@@ -0,0 +1,30 @@
+import { useMutation, useQueryClient } from 'react-query';
+import axios from 'axios';
+
+export default function useDeleteSharedAccess(roomId) {
+  const queryClient = useQueryClient();
+
+  const deleteSharedAccess = (data) => {
+    axios.delete(`/api/v1/shared_accesses/room/${roomId}.json`, { data });
+  };
+
+  const delay = (time) => new Promise((resolve) => {
+    setTimeout(resolve, time);
+  });
```

Comment:
I'd like to spend some more time on this. The only mutations that are needing this are the ones that you are writing - none of the other mutations seem to have this problem

Reply:
Issue was caused by using a arrow function and not including the reserved word `return`
() => { return }